docker/windows/amd64/nanoserver/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Golang binary building stage
+FROM golang:1.9.2
+
+# download the source
+WORKDIR /go/src/github.com/nats-io/prometheus-nats-exporter
+RUN git clone https://github.com/nats-io/prometheus-nats-exporter.git .
+
+# build 
+RUN go build -v -a -tags netgo -installsuffix netgo
+
+# Final docker image building stage
+FROM microsoft/nanoserver
+COPY --from=0 /go/src/github.com/nats-io/prometheus-nats-exporter/prometheus-nats-exporter.exe /prometheus-nats-exporter.exe
+EXPOSE 7777
+ENTRYPOINT ["/prometheus-nats-exporter.exe"]
```

Comment:
I have been told that there should not be `ENTRYPOINT` here (same for windowsservercore dockerfile). Please check https://github.com/nats-io/nats-streaming-docker/commit/9596dfac61688686a6f4955624c559cf101301a9#diff-e2b7dae7d59ab0b01b6fa33dfcc4c7af for details.

Reply:
We need to update this in gnatsd too by the way. I did not get to do it yet.

Reply:
Thanks.  I followed the chain of links and comments that finally lead [here](https://github.com/docker-library/official-images#consistency) - because one can powershell into the images, we need to use CMD.  I'll change this, but as we discussed offline, imo this isn't actually what we want in our images.  If there were such a thing as a scratch that ran with windows, I'd use it here.